### ISSUE TITLE
VRChat error at launch.

### ISSUE THREAD

**Issue by SubDoce** [fholger/openvr_fsr#158]
![image](https://user-images.githubusercontent.com/40150035/194714664-1bdbbe82-183c-4004-971d-656ffb2868dc.png)

I'm facing this error while trying to use the openvr_avi in VRChat. Seems a problem whit the anti-cheat. Anything i can do?

**Comment by haigoux** [comment]
There's not anything you can do about this I believe.

**Comment by quacknathan** [comment]
good ol anticheat... blocks out modifications, that's gonna be a game added to the "does not work" list. EAC blocks any and all client modifications

**Comment by YukiRainee** [comment]
yeah there is nothing you can do about this due to vrchat starting to use easy anticheat if you want you can ask the devs to allow use this openvr api instead but I doubt they would (also sorry for the necro)
